app/src/main/java/com/ataulm/whatsnext/Token.kt

```diff
@@ -1,3 +1,3 @@
 package com.ataulm.whatsnext
 
-data class Token(val accessToken: String, val refreshToken: String, val expiryMillisSinceEpoch: Long)
```

Comment:
We're not using this anywhere - and the codebase is unclear whether this is millis or seconds 😬